aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -169,6 +169,8 @@ def define(cls, spec):
         spec.output(cls.link_label_retrieved, valid_type=orm.FolderData, pass_to_parser=True,
             help='Files that are retrieved by the daemon will be stored in this node. By default the stdout and stderr '
                  'of the scheduler will be added, but one can add more by specifying them in `CalcInfo.retrieve_list`.')
+        spec.exit_code(0, 'NO_ERROR', message='no errors')
+        spec.exit_code(50, 'OUT_OF_MEMORY', message='scheduler reports out of memory')
```

Comment:
The range `100-199` is reserved for generic scheduler/node errors. Let's reserve `100` for the most generic value (if exact reason for failure is unknown) and so maybe use `101` for OOM?